app/javascript/react/screens/App/Plan/helpers.js

```diff
@@ -115,3 +115,23 @@ export const incompleteCancellationTasks = (requestWithTasks, actions, tasksForC
   const completedTasksOfPlan = tasksOfPlan.filter(task => task.state === 'finished');
   return removeCancelledTasksFromSelection(tasksForCancel, completedTasksOfPlan);
 };
+
+export const parseComplexErrorMessages = complexErrorMessage => {
+  const regex1 = RegExp('Conversion host was not found. Download of transformation log aborted.', 'g');
```

Comment:
Nit-pick, take it or leave it, but maybe we should declare each of these regexes above the condition where they are used, so it's easier to correlate them if we add more?

Reply:
> Nit-pick, take it or leave it

Take it! :)
I have moved those RegExp constants in the constants JS